### ISSUE TITLE
Buttons on the “How it works” screens do not have a role of button [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#218]
The “Previous”, “Next“, and “Close” buttons found at the bottom of the “How it works” screens do not convey a role of button. For example, the “Next” button is simply announced as “Next” by TalkBack.

![how-it-works - Previous Next buttons](https://user-images.githubusercontent.com/7034772/91991502-f8500c80-ed00-11ea-8434-9819765275f8.png)

![how-it-works - Close button](https://user-images.githubusercontent.com/7034772/91991599-1584db00-ed01-11ea-8feb-25a5dd0b1b40.png)

We suspect the close ("X") functionality in the top-left of the screen on the "How it works" screens, is a graphical "X" button, lacking an accessible role or label, which results in TalkBack not conveying any information when it receives focus (nothing is announced). 

![how-it-works - close x button](https://user-images.githubusercontent.com/7034772/91991432-deaec500-ed00-11ea-847f-634ab62f4906.png)

**Remediation Guidance:**

We recommend using the `Button` element for all elements that perform discrete functions, like "Previous", "Next" and "Close".
```
<Button
   accessibilityLabel="Previous."
   accessibilityRole="button"
/>
```

Severity: High

**Comment by floridemai** [comment]
@jonnyjames This has now been addressed

**Comment by jonnyjames** [comment]
I can confirm this issue is fixed. Closing this issue.
